### ISSUE TITLE
Minor CSS issue on mobile for preview button

### ISSUE THREAD

**Issue by iamrobert** [FLEXIcontent/flexicontent-cck#268]
Hi - the following issue presents itself with mobile and the 32px for preview button:
![issue](https://cloud.githubusercontent.com/assets/8617673/9338423/4155a6f6-4617-11e5-8106-1c1b842fe9a0.png)

Fix:

     @media (max-width: 767px) {
	#toolbar .preview.btn {margin-left: 0 !important;}
      }

**Comment by ggppdk** [comment]
I think, it is best to do the reverse,
add a CSS class (spaced-btn),

-- that will apply margin-left only if screen is large enough:

```@media (min-width: 1020px) {
    .btn.spaced-btn { margin-left:32px; }
}```

Fixed with 8949c7d

**Comment by iamrobert** [comment]
Yes - its better your way
